### DISCUSSION TITLE
管理员的账号密码如何修改？ 在 gitserver 端改吗？

### DISCUSSION THREAD

**Discussion by karypi** [OpenCSGs/csghub#600] [Q&A]

**Comment by HaiHui886** [comment]
指的是管理员的初始化密码吗？

**Comment by karypi** [comment]
> 指的是管理员的初始化密码吗？

对

**Comment by HaiHui886** [comment]
请参考这个casdoor/conf/init_data.json初始化配置文件，可以进行修改。
https://github.com/OpenCSGs/CSGHub/blob/3699f671d6ac8aa00449cb47bd68abbcdc819100/deploy/all_in_one/casdoor/conf/init_data.json#L156

**Comment by karypi** [comment]
> 请参考这个casdoor/conf/init_data.json初始化配置文件，可以进行修改。
> 
> https://github.com/OpenCSGs/CSGHub/blob/3699f671d6ac8aa00449cb47bd68abbcdc819100/deploy/all_in_one/casdoor/conf/init_data.json#L156

casdoor 这个是新的组件吗？  我的版本里没有在这个组件。  我要修改的是 admin001 的默认密码。https://github.com/OpenCSGs/CSGHub/blob/main/docs/setup_en.md#default-user-on-system-initialization

**Comment by karypi** [comment]
> 请参考这个casdoor/conf/init_data.json初始化配置文件，可以进行修改。
> 
> https://github.com/OpenCSGs/CSGHub/blob/3699f671d6ac8aa00449cb47bd68abbcdc819100/deploy/all_in_one/casdoor/conf/init_data.json#L156

你们这个服务还在维护么...

**Comment by HaiHui886** [comment]
是的，最新版都在casdoor中维护用户
